spec.html

```diff
@@ -22,10 +22,17 @@ <h1>Introduction</h1>
   <h1>Syntax</h1>
 
   <emu-grammar>
-      LogicalORExpression[In, Yield, Await] :
-        LogicalANDExpression[?In, ?Yield, ?Await]
-        LogicalORExpression[?In, ?Yield, ?Await] `||` LogicalANDExpression[?In, ?Yield, ?Await]
-        <ins>LogicalORExpression[?In, ?Yield, ?Await] `??` LogicalANDExpression[?In, ?Yield, ?Await]</ins>
+      ConditionalExpression[In, Yield, Await] :
+        <del>LogicalORExpression[?In, ?Yield, ?Await]</del>
+        <del>LogicalORExpression[?In, ?Yield, ?Await] `?` AssignmentExpression[+In, ?Yield, ?Await] `:` AssignmentExpression[?In, ?Yield, ?Await]</del>
```

Comment:
I think these two lines can be dropped, we only need `NullishExpression` here.

Reply:
They're inside `<del>` tags...

Reply:
Do you mean just not showing them in the diff?

I was about to comment that basically everywhere in the spec that `LogicalORExpression` occurs requires an update, so it would be good to include them here, even if they're all just routine additions.